usr/share/rear/skel/NBKDC/etc/init/rcmd-executor.conf

```diff
@@ -0,0 +1,8 @@
+description "NBK DC Remote Command Executor"
+start on runlevel [2345]
+stop on runlevel [016]
+
+respawn
+normal exit 0
+
+exec /opt/NovaStor/DataCenter/rcmd-executor/rcmd-executor run
```

Comment:
Why is the program location hard coded here? Maybe you also hard-code it in the other scripts? Will save you the `get_nbkdc_dir` function and make your code much simpler.